steps/assess.ts

```diff
@@ -111,12 +111,14 @@ export const provideRequirements = async (page: Page) => {
   await requirementsPage.clickSubmit()
 }
 
-export const makeDecision = async (page: Page) => {
+export const makeDecision = async (page: Page, acceptApplication = true) => {
```

Comment:
could you make the second parameter an object please?  
I appreciate it's a little dogmatic but I think having options as an object makes it easier to read, even when there's only one option.